p2p/host/relayv2/relay.go

```diff
@@ -0,0 +1,91 @@
+package relayv2
```

Comment:
let's call this package `relaysvc` or something similar? We are moving the autorelay stuff in `autorelay` anyway.